src/webhint-theme/source/components/treeview/treeview.js

```diff
@@ -169,12 +169,12 @@
 
     var toggleExpandMobileMenu = function (evt) {
         var menu = evt.currentTarget.querySelector('#mobile-toc-submenu');
-        var isClosed = menu.getAttribute('aria-hidden') === 'true';
+        var isClosed = menu.getAttribute('hidden') === '';
```

Comment:
For boolean attributes any value is effectively `true`, even if I write `hidden="hidden"`.
```suggestion
        var isClosed = menu.hasAttribute('hidden');
```